src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorComponentSearchEngine.cs

```diff
@@ -149,7 +149,7 @@ internal class DefaultRazorComponentSearchEngine : RazorComponentSearchEngine
         return null;
     }
 
-    private static StringSegment RemoveGenericContent(StringSegment typeName)
+    public static StringSegment RemoveGenericContent(StringSegment typeName)
```

Comment:
Note: now that this is being used multiple places, should it be extracted to some utility class?

Reply:
This is definitely a weird spot for a utility function, so nice work finding it, but at the same time I don't think there is any other obvious spot for it either. I don't think its a big deal staying here for now.